catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -143,20 +144,20 @@ protected Object executeWithSubject() throws Exception {
         final File outputFile = initOutputFile(output);
         if (outputFile.exists()) {
             printErrorMessage(String.format("File [%s] already exists!", outputFile.getPath()));
-            return null;
```

Comment:
No runtime impact other than it exits the command errorfully instead of with zero information about what went wrong.